src/components/ModiteListItem/index.tsx

```diff
@@ -26,19 +25,14 @@ const ModiteListItem: FunctionComponent<ListItemProps & RouteComponentProps> = (
   };
 
   return (
-    <IonMenuToggle key={modite.id} auto-hide="false">
-      <IonItem button onClick={handleItemClick}>
-        <IonThumbnail aria-hidden="true" slot="start" class={s.thumbnailContainer}>
-          <ModiteImage modite={modite} />
-        </IonThumbnail>
-
-        <IonLabel>{modite.real_name}</IonLabel>
-        <IonLabel aria-hidden="true" class={s.tod}>
-          {modite.tod}
-        </IonLabel>
-        <IonLabel class={s.time}>{modite.localTime}</IonLabel>
-      </IonItem>
-    </IonMenuToggle>
+    <div className={s.itemInnerCt} onClick={handleItemClick}>
```

Comment:
To make a `div` with `onClick` accessible, you need to either:
* use `button` instead, or
* add ARIA markup such as `role="button"`, hopefully with managed `aria-pressed` state

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
fixed